roles/shared_handlers/handlers/main.yml

```diff
@@ -0,0 +1,26 @@
+# This role holds reusable handlers that can be included in multiple playbooks to keep things DRY
+- name: precompile assets
```

Comment:
I'm wondering if this should be a task as well.

Reply:
Interesting. I thought a rollback is restoring the old files including the assets. So it doesn't need to compile assets... Anyway, I'm happy to keep it as handler. I just like your approach so much. :-)

Reply:
That's a good point! The original rollback role used `notify: precompile assets` so I didn't remove it, but maybe it can be striped out. I think updating unicorn should be enough.